.travis/deploy_docker.sh

```diff
@@ -13,15 +13,16 @@ cd $TRAVIS_BUILD_DIR
 project_version=$(sbt "project common" version -Dsbt.log.noformat=true | perl -ne 'print "$1\n" if /info.*(\d+\.\d+\.\d+[^\r\n]*)/' | tail -n 1 | tr -d '\n')
 if [[ "${tag}" = *"${project_version}" ]]; then
     sbt "project loader" docker:publishLocal
+    sbt "project streamloader" docker:publishLocal
     sbt "project mutator" docker:publishLocal
     sbt "project repeater" docker:publishLocal
     sbt "project forwarder" docker:publishLocal
     docker push "${docker_repo}/snowplow/snowplow-bigquery-mutator:${tag}"
     docker push "${docker_repo}/snowplow/snowplow-bigquery-loader:${tag}"
+    docker push "${docker_repo}/snowplow/snowplow-bigquery-streamloader:${tag}"
     docker push "${docker_repo}/snowplow/snowplow-bigquery-repeater:${tag}"
     docker push "${docker_repo}/snowplow/snowplow-bigquery-forwarder:${tag}"
```

Comment:
Yeah, maybe we should just delete it completely from working tree. But while it's in the worktree - I don't see any harm for it being updated with latest Beam/Scio in case some lost soul still uses it.

Reply:
+1 to completely remove it given that it's deprecated. I created https://github.com/snowplow-incubator/snowplow-bigquery-loader/issues/121